source/Sashimi/AzureWebAppServiceMessageHandler.cs

```diff
@@ -63,9 +65,27 @@ public AzureWebAppServiceMessageHandler(ILog logger)
                 endpoint.WebAppSlotName = webAppSlotName;
             }
 
+            endpoint.DefaultWorkerPoolId = GetWorkerPoolId(messageProperties, variables, workerPoolIdResolver);
+
             return endpoint;
         }
 
+        string GetWorkerPoolId(IDictionary<string, string> messageProperties, VariableDictionary variables, Func<string, string> workerPoolIdResolver)
```

Comment:
Could we make this return `string?` and return null when it's isn't found? I think that'd be more consistent with what we do elsewhere. DefaultWorkerPoolId should end up null if it hasn't been set, right?